docs/id/docs/index.md

```diff
@@ -410,58 +410,59 @@ Try changing the line with:
         ... "item_price": item.price ...
 ```
 
-...and see how your editor will auto-complete the attributes and know their types:
+...dan lihatlah bagaimana editormu akan melakukan penyelesaian otomatis atribut-atribut dan tahu masing-masing tipenya.
 
 ![editor support](https://fastapi.tiangolo.com/img/vscode-completion.png)
 
-For a more complete example including more features, see the <a href="https://fastapi.tiangolo.com/tutorial/">Tutorial - User Guide</a>.
+Untuk contoh yang lebih kompleks termasukfitur-fitur yang lain, lihatlah <a href="https://fastapi.tiangolo.com/tutorial/">Tutorial - Pedoman Pengguna</a>.
 
-**Spoiler alert**: the tutorial - user guide includes:
+**Spoiler alert**: Tutorial - pedoman pengguna meliputi:
 
 * Declaration of **parameters** from other different places as: **headers**, **cookies**, **form fields** and **files**.
-* How to set **validation constraints** as `maximum_length` or `regex`.
-* A very powerful and easy to use **<abbr title="also known as components, resources, providers, services, injectables">Dependency Injection</abbr>** system.
-* Security and authentication, including support for **OAuth2** with **JWT tokens** and **HTTP Basic** auth.
-* More advanced (but equally easy) techniques for declaring **deeply nested JSON models** (thanks to Pydantic).
-* Many extra features (thanks to Starlette) as:
+* Deklarasi dari **parameter-parameter** dari tempat yang berbeda-beda seperti: **headers**, **cookies**, **form fields** dan **files**.
+* Bagaimana menge-set **batasan validasi** sebagai `maximum_length` atau `regex`.
+* System yang sangat tangguh dan mudah digunakan **<abbr title="juga dikenal sebagai components, resources, providers, services, injectables">Dependency Injection</abbr>** system.
+* Keamanan dan otentifikasi, termasuk support untuk **OAuth2** dengan **JWT tokens** dan **HTTP Basic** auth.
+* Teknik-teknik yang lebih maju (tetapi sama-sama mudah) untuk mendeklarasikan **model-model JSON bersarang** (terima kasih untuk Pydantic).
```

Comment:
how about 'Teknik-teknik lanjutan' ?

Reply:
I used "Teknik-teknik **yang lebih maju**" to explain "**More** advanced techniques".